modules/systemd-journal/journal-reader.c

```diff
@@ -725,6 +740,7 @@ journal_reader_options_init(JournalReaderOptions *options, GlobalConfig *cfg, co
           options->prefix = g_strdup(default_prefix);
         }
     }
+  options->all_namespaces = TRUE;
```

Comment:
From the point of work it is equivalent, but this looks more like a "default value" than an initialization step. Consider putting it in `journal_reader_options_defaults`